circuitClient.js

```diff
@@ -122,6 +124,68 @@ class CircuitClient {
       }]);
     });
   }
+  
+  async setMyStatusMessage(statusMessage){
+    return this.client.setStatusMessage(statusMessage);
+  }
+
+  getMyStatusMessage(){
+    return this.client.getStatusMessage();
+  }
+
+  async setPresenceAvailable(){
+    const statusMsg = await this.client.getStatusMessage();
+
+    return this.client.setPresence({
+      state: Circuit.Enums.PresenceState.AVAILABLE,
+      statusMessage: statusMsg
+    });
+  }
+  
+  async setPresenceDnd(untilTime, duration){
+
+    const statusMsg = await this.client.getStatusMessage();
+
+    if(untilTime !== ""){
+      untilTime = new Date(untilTime).getTime();
+      
+      return this.client.setPresence({
+      state: Circuit.Enums.PresenceState.DND,
+      dndUntil: untilTime,
+      statusMessage: statusMsg
+      });
+    }
+    else if(duration != ""){
```

Comment:
Move "else if" to line above to follow our styleguide.
Also use single quotes rather than double quotes.